### PR TITLE
chore: Fix execute not start vt issue.

### PR DESCRIPTION
**Pull request by He-Pin**
Motivation:
In my last pr, I forgot to start the VT :(

Result:
Fixed :)

### CODE REVIEW

File: jvm/src/main/scala/async/VThreadSupport.scala [hunk 1]
```diff
@@ -10,7 +10,9 @@ object VThreadScheduler extends Scheduler:
     .name("gears.async.VThread-", 0L)
     .factory()
 
-  override def execute(body: Runnable): Unit = VTFactory.newThread(body)
+  override def execute(body: Runnable): Unit =
+    val th = VTFactory.newThread(body)
+    th.start()
```
Comment (He-Pin) [inline jvm/src/main/scala/async/VThreadSupport.scala:15]: @m8nmueller I forgot to start it, sorry~~

**Comment by natsukagami** [comment]
Can you add a test to make sure things work as expected? 
I could also add one if wanted.

**Comment by He-Pin** [comment]
Sorry, I still can't build the snapshot out locally, and a little busy at work. cherry pick if you like. Sorry for missed the `start` call in my last pr.

**Comment by He-Pin** [comment]
I'm using Windows 11, so I didn't

**Comment by natsukagami** [comment]
Sorry for the force-push, I wanted to rebase it to the latest upstream main branch

**Comment by He-Pin** [comment]
@natsukagami Thanks I want to learn how to force push to a contributor branch too

**Comment by natsukagami** [comment]
> force push to a contributor branch 

As long as "Maintainers are allowed to edit this pull request." I think regular `git push -f` works. You need to set the correct upstream for the branch, but if you have GitHub cli it's easy to have it automate all the setting up for you: `gh pr checkout 39`.

**Comment by He-Pin** [comment]
@natsukagami I can't install the github cli, company doesn't allow it. thank you ,just learn something new

**Pull request merged**